src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -72,10 +72,11 @@ function validateExplicitKey(element, parentType) {
   warning(
     false,
     'Each child in an array or iterator should have a unique "key" prop.' +
-    '%s%s%s',
+    '%s%s%s%s',
     addenda.parentOrOwner || '',
     addenda.childOwner || '',
-    addenda.url || ''
+    addenda.url || '',
```

Comment:
We shouldn't need these other addenda now since the stack should cover it. Can you remove the code for them? We still will want to dedupe the warnings based on owner though as we're currently doing.